co-log-core.cabal

```diff
@@ -41,13 +41,14 @@ tested-with:         GHC == 8.2.2
                      GHC == 9.0.2
                      GHC == 9.2.1
                      GHC == 9.2.2
+                     GHC == 9.4.2
```

Comment:
For the full `9.4` support that would be nice to also add  the check on CI. This file would need to be patched for that 🙂 

* https://github.com/co-log/co-log-core/blob/main/.github/workflows/ci.yml

Reply:
Done.